src/app/components/wrapper-components/tutorial-game-wrapper/tutorial-game-wrapper.wrapper.component.spec.ts

```diff
@@ -1074,4 +1075,17 @@ describe('TutorialGameWrapperComponent (wrapper)', () => {
             expect(wrapper.stepFinished[wrapper.stepIndex]).toBeFalse();
         }));
     });
+    describe('getConfig', () => {
+        it('should provide initial default config to game component', fakeAsync(async() => {
+            // Given any tutorial
```

Comment:
given any tutorial for a game that has a specific default config
et mettre le spyOn dans le given plutôt que le when?